thesis.tex

```diff
@@ -1,190 +1,154 @@
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\section{Main Ideas}
-\label{sec:mainideas}
-\input{sections/main-ideas.tex}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% Technical Meat
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\section{Technical Meat}
-\label{sec:technical-meat}
-\input{sections/technical-meat.tex}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-% Discussion and Outlook
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\section{Discussion and Outlook}
-\label{sec:discussion}
-\input{sections/discussion.tex}
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%% Bibliography
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\cleardoublepage
-
-\bibliography{bibliography} % link bibliography.bib
-
-
-\cleardoublepage
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%% Selbständigkeitserklärung
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\thispagestyle{empty}
-\section*{Selbständigkeitserklärung}
-
-Hiermit versichere ich, dass ich die vorliegende Masterarbeit
-selbständig und nur mit den angegebenen Hilfsmitteln angefertigt habe und dass alle Stellen, die dem Wortlaut oder dem
-Sinne nach anderen Werken entnommen sind, durch Angaben von Quellen als
-Entlehnung kenntlich gemacht worden sind.
-Diese Masterarbeit wurde in gleicher oder ähnlicher Form in keinem anderen
-Studiengang als Prüfungsleistung vorgelegt.
-
-\vskip 3cm
 
-Ort, Datum	\hfill Unterschrift \hfill
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-%%% Ende
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\frontmatter
+
+  \include{src/title}
+  % ensure title pages do not advance the pager counter
+  \setcounter{page}{1}
+  \include{src/abstract}
+
+  \tableofcontents
+
+\mainmatter
+
+  % delete me
+  \include{usage}
+
+  \include{src/introduction}
+  \include{src/overview}
+  \include{src/idea}
+  \include{src/evaluation}
+  \include{src/discussion}
+  \include{src/related_work}
+  \include{src/conclusion}
+
+\backmatter
+  \printbibliography
+
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  %%% Selbständigkeitserklärung
+  %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+  \thispagestyle{empty}
+  \begin{otherlanguage}{ngerman}
+  \chapter*{Selbständigkeitserklärung}
+  Hiermit erkläre ich, dass ich diese schriftliche Abschlussarbeit selbständig verfasst habe, keine
+  anderen als die angegebenen Hilfsmittel und Quellen benutzt habe und alle wörtlich oder
+  sinngemäß aus anderen Werken übernommenen Aussagen als solche gekennzeichnet habe.
+  
+  \vspace{1cm} 
+  \noindent
+  \hrulefill \hspace{2.5cm} \hrulefill
+  
+  \noindent
+  Ort, Datum \hfill Unterschrift
+  \end{otherlanguage}
```

Comment:
Yes, I noticed that wording is different. I have directly copied it from the official thesis [registration form](https://uni-tuebingen.de/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/studium/studierende/downloads/informationen-und-formulare/) since it was the most "official" one I could find.